docs/alerts.rst

```diff
@@ -0,0 +1,242 @@
+Alerts
+======
+
+Use alerts to get notifications about important App Control-monitored activities, such as the
+appearance or spread of risky files on your endpoints. The Carbon Black Cloud Python SDK provides
+all of the functionalities you might need to use them efficiently.
+You can use all of the operations shown in the API such as retrieving, filtering, dismissing, creating and updating.
+The full list of operations and attributes can be found in the :py:mod:`BaseAlert() <cbc_sdk.platform.alerts.BaseAlert>` class.
+
+For more information see
+`the developer documentation <https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alerts-api/>`_
+
+Retrieving of Alerts
+--------------------
+
+With the example below, you can retrieve alerts with the minimum severity of ``7``.
+
+.. code-block:: python
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import BaseAlert
+    >>> api = CBCloudAPI(profile='sample')
+    >>> alerts = api.select(BaseAlert).set_minimum_severity(7)[:5]
+    >>> print(alerts[0].id, alerts[0].device_os, alerts[0].device_name, alerts[0].category)
+    d689e626-5d6a-<truncated> WINDOWS Alert-WinTest THREAT
+
+
+Filtering
+^^^^^^^^^
+
+Filter alerts using the fields described in the
+`Alert Search Schema <https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alerts-api/#alert-search>`_.
+
+You can use the ``where`` method to filter the alerts. The ``where`` supports strings as well as ``solrq`` query objects
```

Comment:
I totally forgot that solrq was a sub package we included to support this functionality. The more common way we see the where function used is with query strings like `category:THREAT or category:INFO` also we don't support `INFO` so this guide needs updated according to the bug that was opened.